get_images.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
+
+"""
+    images.py
+
+    MediaWiki Action API Code Samples
+    Demo of `Images` module: Sending GET request to images
```

Comment:
You could elaborate the description a bit, something like "Send get request to obtain images embedded on a page.."

Reply:
Text now reads 

> Demo of `Images` module: Send a GET request to obtain a JSON
    object listing all of the image files embedded on a single
    page.

in renamed file